### PR TITLE
google-auth-httplib2-feedstock 0.2.0 :snowflake: 

### PR DESCRIPTION
**Pull request by Andrii-Osipov**
google-auth-httplib2-feedstock 0.2.0 :snowflake: 

**Destination channel:** Snowflake

- [PKG-6076](https://anaconda.atlassian.net/browse/PKG-6076) 
- [google-auth-library-python-httplib2](https://github.com/GoogleCloudPlatform/google-auth-library-python-httplib2)
- [PyPi](https://pypi.org/project/google-auth-httplib2/)
- [conda-forge](https://github.com/conda-forge/google-auth-httplib2-feedstock)

[PKG-6076]: https://anaconda.atlassian.net/browse/PKG-6076?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by Andrii-Osipov** [comment]
I will remove this channel from abs.yml https://staging.continuum.io/prefect/fs/httplib2-feedstock/pr2/5500118
after this pr is merged:https://github.com/AnacondaRecipes/httplib2-feedstock/pull/2.

**Review by scotthain — APPROVED** [review]
LGTM

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,37 +1,45 @@
...
     - pip
+    - wheel
+    - setuptools
   run:
-    - python >=3.6
+    - python
     - google-auth
-    - httplib2 >=0.15.0
+    - httplib2 >=0.19.0
     - six
```
Comment (cbouss) [inline recipe/meta.yaml]:
```suggestion
```
six is not listed here: https://github.com/googleapis/google-auth-library-python-httplib2/blob/v0.2.0/setup.py#L37

File: recipe/meta.yaml [hunk 2]
```diff
@@ -1,37 +1,45 @@
...
-  sha256: a07c39fd632becacd3f07718dfd6021bf396978f03ad3ce4321d060015cc30ac
+  url: https://github.com/googleapis/google-auth-library-python-httplib2/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 7e714ff7a4cd6c69b76f8c91dfe177cdce49ae8bfb398b22a7c1468f3915037d
 
 build:
   number: 0
-  noarch: python
-  script: {{ PYTHON }} -m pip install . -vv
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  skip: True  # [py<39]
```
Comment (cbouss) [inline recipe/meta.yaml]:
```suggestion
  skip: True  # [py<36]
```
https://github.com/googleapis/google-auth-library-python-httplib2/blob/v0.2.0/setup.py#L42

File: recipe/meta.yaml [hunk 3]
```diff
@@ -1,37 +1,45 @@
...
   imports:
     - google_auth_httplib2
   commands:
     - pip check
+    - pytest -v tests
   requires:
     - pip
+    - pytest
+    - flask
+    - pytest-cov
```
Comment (cbouss) [inline recipe/meta.yaml]:
```suggestion
```
Not needed.

**Review by cbouss — COMMENTED** [review]
Some minor comments

**Review by cbouss — APPROVED** [review]

**Pull request merged**